src/actions/admin.test.js

```diff
@@ -805,23 +805,23 @@ describe('Actions.Admin', () => {
         const downloadUrl = 'testplugin.tar.gz';
         const testPlugin = {id: 'testplugin', webapp: {bundle_path: '/static/somebundle.js'}};
 
-        let urlMatch = `/plugins/install_from_url?plugin_download_url=${downloadUrl}&force=false`;
+        let urlMatch = `/plugins/install_from_url?download_url=${downloadUrl}&signature_url=&force=false`;
```

Comment:
@ali-farooq0 same comment as I mention in @iomodo server changes:

`plugin_download_url` has been released in 5.14 though. It's already documented [here](https://api.mattermost.com/#tag/plugins%2Fpaths%2F~1plugins~1install_from_url%2Fpost). This would break backward compatibility.

Reply:
yep reverting it back to preserve backwards compatibility.